Purchases/Caching/RCDeviceCache.h

```diff
@@ -33,21 +33,21 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)cachePurchaserInfo:(NSData *)data forAppUserID:(NSString *)appUserID;
 
-- (BOOL)isPurchaserInfoCacheStale;
+- (BOOL)isPurchaserInfoCacheStaleForAppUserID:(NSString *)appUserID isAppBackgrounded:(BOOL)isAppBackgrounded;
```

Comment:
not a huge fan of how this API is turning out, it feels overly complicated. But I didn't want to perform a major refactor for this.